webplugin/js/app/events/applozic-event-handler.js

```diff
@@ -58,6 +58,11 @@ Kommunicate.KmEventHandler = {
         }
     },
     onMessageSent: function (message) {
+        window.kmWidgetEvents.sendEventToGoogleAnalytics(
```

Comment:
is window. required?
see how else we can do it
some places we have kommunicateUI. and Kommunicate. as welll